executor/builder.go

```diff
@@ -1552,7 +1552,7 @@ func buildNoRangeTableReader(b *executorBuilder, v *plan.PhysicalTableReader) (*
 	if containsLimit(dagReq.Executors) {
 		e.feedback = statistics.NewQueryFeedback(0, nil, 0, ts.Desc)
 	} else {
-		e.feedback = statistics.NewQueryFeedback(ts.Table.ID, ts.Hist, int64(ts.StatsCount()), ts.Desc)
+		e.feedback = statistics.NewQueryFeedback(e.tableID, ts.Hist, int64(ts.StatsCount()), ts.Desc)
```

Comment:
Can we use table.GetPhysicalID()?

Reply:
The refactor PR is not in the master yet, so there is no `GetPhysicalID()`